src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -0,0 +1,645 @@
+  <data name="Profile_FilledBoxCursorShape.Content" xml:space="preserve">
+    <value>Filled box</value>
+  </data>
+  <data name="Profile_FontFace.Header" xml:space="preserve">
+    <value>Font face</value>
+  </data>
+  <data name="Profile_FontFace.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Name of the font face used in the profile.</value>
+  </data>
+  <data name="Profile_FontSize.Header" xml:space="preserve">
+    <value>Font size</value>
+  </data>
+  <data name="Profile_FontSize.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Size of the font in points.</value>
+  </data>
+  <data name="Profile_FontWeight.Header" xml:space="preserve">
+    <value>Font weight</value>
+  </data>
+  <data name="Profile_FontWeight.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Sets the weight (lightness or heaviness of the strokes) for the given font.</value>
+  </data>
+  <data name="Profile_ForegroundColor.Header" xml:space="preserve">
+    <value>Foreground color</value>
+  </data>
+  <data name="Profile_ForegroundColorToolTip.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Sets the text color. Overrides the foreground from the color scheme.</value>
+  </data>
+  <data name="Profile_General.Header" xml:space="preserve">
+    <value>General</value>
+  </data>
+  <data name="Profile_GracefulCloseOnExit.Content" xml:space="preserve">
+    <value>Graceful</value>
+  </data>
+  <data name="Profile_GrayscaleAntialiasingMode.Content" xml:space="preserve">
+    <value>Grayscale</value>
+  </data>
+  <data name="Profile_Hidden.Content" xml:space="preserve">
+    <value>Hide profile from dropdown</value>
+  </data>
+  <data name="Profile_Hidden.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>If checked, the profile will not appear in the list of profiles. This can be used to hide default profiles and dynamically generated profiles, while leaving them in your settings file.</value>
+  </data>
+  <data name="Profile_HiddenScrollbarVisibility.Content" xml:space="preserve">
+    <value>Hidden</value>
+  </data>
+  <data name="Profile_HistorySize.Header" xml:space="preserve">
+    <value>History size</value>
+  </data>
+  <data name="Profile_HistorySize.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>The number of lines above the ones displayed in the window you can scroll back to.</value>
+  </data>
+  <data name="Profile_Icon.Header" xml:space="preserve">
+    <value>Icon</value>
+  </data>
+  <data name="Profile_Icon.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Image file location of the icon used in the profile. Displays within the tab and the dropdown menu.</value>
+  </data>
+  <data name="Profile_IconBrowse.Content" xml:space="preserve">
+    <value>Browse</value>

```

Comment:
I think the localizers can handle it. It's pretty annoying t oneed duplicates, but it's even more annoying to do the gymnastics required to unify them (since they both have to be the same element type so you can use `A.Content` as the key, or you need to use `RS_` from code

Reply:
Ah, I mostly just meant it so that we'd always use "Browse..." as opposed to "Browse". I'm thinking we need the "..." to imply that clicking the button will open another dialog. I get the `A.Content` thing though, so I guess we'll just need to be diligent on our own.

Reply:
Ah. Yes, `...` for all.